diamm/templates/aboutpages_menu.jinja2

```diff
@@ -0,0 +1,18 @@
+{% set pages, subpages = aboutpages_list() %}
+
+ <ul class="column">
```

Comment:
I would be careful about using the 'column' class, since this is defined in Skeleton. If you were using the column class you generally will need to specify the number of columns, so 'sixteen columns'. It would probably be best if you made your own class, like 'about-menu-columns' to differentiate it.